### ISSUE TITLE
Training on multiple samples

### ISSUE THREAD

**Issue by jsnelgro** [tomlepaine/fast-wavenet#12]
Hi, I'm pretty new to machine learning, so maybe this is a silly question, but I was wondering how I would train this network using more than just one sample? Because it looks like the `make_batch` function only loads a single file. Also, I'm not sure if this is related, but would I need to train a separate network for each class of generator? Or is there a way to label the training data? Any help and tutoring is much appreciated!

**Comment by tomlepaine** [comment]
Hi @jsnelgro 
This code is just an example implementation of the **fast wavenet generation algorithm**. We have since ported the algorithm to [tensorflow-wavenet](https://github.com/ibab/tensorflow-wavenet). The repo is meant for training on multiple samples, so please use that 😺 

For a tutorial, have you seen the [tensorflow tutorials](https://www.tensorflow.org/tutorials/)? That should provide examples of sampling random mini batches from a dataset and using it to train a model using minibatch gradient descent. Its a bit beyond the scope of this repo.

Also if you want to generate different classes of audio, you could have a model for each class, or **condition** on the class. See the [wavenet paper](https://arxiv.org/pdf/1609.03499v2.pdf) for more details.

**Comment by jsnelgro** [comment]
awesome, thanks for the resources @tomlepaine!
